core/src/main/scala/org/apache/spark/ui/jobs/StagePage.scala

```diff
@@ -132,7 +132,7 @@ private[ui] class StagePage(parent: StagesTab, store: AppStatusStore) extends We
     val totalTasksNumStr = if (totalTasks == storedTasks) {
       s"$totalTasks"
     } else {
-      s"$storedTasks, showing ${totalTasks}"
+      s"$totalTasks, showing $storedTasks"
```

Comment:
Could you update the title to `[SPARK-25503][CORE][WEBUI] ...`?

Reply:
Done. Thanks